omnilib/changelog.txt

```diff
@@ -1,3 +1,8 @@
+Version: 3.18.8
+Date: XXX
+  Features:
+    - Added steam powered buildings
+>>>>>>> Steam omnitractor and omniphlog added
```

Comment:
Will break the changelog functionality inside factorio (>>>>> seems like a merge issue)